The "main" entry point is missing

Hello and thank you for your work.

When trying to include one of your rules in my TSLint setup, my build fails because of what I believe to be a bug.

```ts
Cannot find module '<path/to/my/project>/node_modules/@crazyfactory/tslint-rules/lib/index.js'. Please verify that the package.json has a valid "main" entry
```

When we look at the contents of `node_modules`, the `lib/index.js` file is missing indeed.

![image](https://user-images.githubusercontent.com/20233319/66901920-f5bcc980-efff-11e9-82ad-bcd507d0ca40.png)

However, it should be there as declared in `package.json`.

```
"main": "./lib/index.js",
```

I believe the solution is to put a [barrel file](https://basarat.gitbooks.io/typescript/docs/tips/barrel.html) in `src` so it's preserved in the transpiled output. If you agree, I can take submit a PR doing exactly that.

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@crazyfactory/tslint-rules",
   "private": false,
   "description": "",
-  "main": "./lib/index.js",
+  "main": "tslint-rules.json",
   "config": {
     "validate-commit-msg": {
       "types": [

--- a/tslint-rules.json
+++ b/tslint-rules.json
@@ -1,0 +1,8 @@
+{
+  "rulesDirectory": "./lib",
+  "rules": {
+    "import-react": [true, {"type": "star"}],
+    "hex-format": true,
+    "interface-sort-keys": true
+  }
+}

